Documentation/Signoffs/CommunicationsSignoff.md

```diff
@@ -0,0 +1,38 @@
+# Communication Signoff
+## FUNCTION OF THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Communication subsystem is responsible for receiving data from the Microcontroller subsystem, connecting the device to the Internet, and formatting and sending the received data to the Web & Wireless subsystem.  This subsystem will utilize 4G LTE (Long-term Evolution) as a standard in telecommunications to connect to the Internet for its near global coverage.  
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  The constraints given below are generalized to allow for comparison of any potential contender for the solution.  Constraints on the subsystem are as follows:
+1. The component must be able to utilize 4G LTE telecommunication.  4G LTE operating frequencies include 600 MHz, 700 MHz, 1700/2100 MHz, 2300 MHz, and 2500 MHz bands.  This is the chosen method of accessing the Internet due to its coverage and support in place.
+2. The component must have a slot and support for a Subscriber Identity Module card (SIM).  The component must be able to use this SIM card and its identifying information to access the specified broadband cellular network related to the SIM card's service provider.
+3. Price for this component should remain under $100 as end-users only have a budget of $300.  Allocating 1/3 of the budget is from estimating the costs of other subsystems and viewing prices of available options that satisfy the functions.
+4. For receiving data from the Microcontroller subsystem, Serial Peripheral Interface (SPI) or Universal Asynchronous Receiver/Transmitter (UART) as standards for hardware communication will need to be achievable.  
+5. There must be sufficient memory for the code. Comparing similar operating code (Wireless Sensor Collection and Small-time Processes Operating) with libraries and compilation show a range of 12 KB (Kilobyte) (10%) above or below 120 KB in flash memory. To account for worst case, 132 KB is the required memory.  
```

Comment:
Not clear where this comes from. Does this subsystem have its own processor?

Reply:
There will be a some coding involved and stored on the board.